### PR TITLE
Add ebuild for enlightenment 0.20.2

### PR DESCRIPTION
**Pull request by dirk-olmes**
Now that e20 is stable, I updated the ebuild's version number and dependency minimum version requirements.

Would you like to include this ebuild?

### CODE REVIEW

**Comment by pavlix** [comment]
@okias Since I abandoned the enlightenment packages, it would be nice to have someone to take care of those in ixit, or to start a new repository just for enlightenment. I'm willing to help someone in the latter case.

**Comment by pavlix** [comment]
@dirk-olmes As far as I know @okias is ok with keeping and improving enlightenment ebuilds in *ixit* overlay. So the only issue is that I originally added those ebuilds and kept them as long as I was actively using enlightenment myself with help of other users. I'm no longer keeping it and my personal opinion is that we'd need at least one person actively maintaining those ebuilds and acking/accepting pull requests from other people. In that case it remains a question whether it's better to move enlightenment in a separate repository and maintain it there or keep it part of ixit.

**Comment by dirk-olmes** [comment]
@pavlix I'd be willing to maintain those ebuilds. Note that  I did not modify your original ebuild much - it worked fine for all the updates I submitted. So I cannot really claim that I'd be very deep into the whole enlightenment ebuild either.

Going forward I'd vote for a separate overlay just for clearer separation of concerns. I had to carefully review changes to other installed packages that came in via the ixit overlay in the past.

**Comment by rafspiny** [comment]
@pavlix @dirk-olmes I would be happy to help in maintaining the Enlightenment packages for Gentoo.
I submitted some patch files for Enlightenment 0.21.2 on [bugs.gentoo.org](https://bugs.gentoo.org/show_bug.cgi?id=582482).

Thanks

**Comment by pavlix** [comment]
@dirk-olmes @rafspiny I'm happy to see someone's willing to take over.

**Comment by pavlix** [comment]
So I talked to @rafspiny and we decided to create a new overlay.

https://github.com/rafspiny/enlightenment-live

Anyone who wants to be part of this effort, please talk to @rafspiny over github or #gentoo-e channel on Freenode and/or use the above github project to start issues and pull requests. This pull request will be closed and the ebuild added to enlightenment-live.

**Comment by okias** [comment]
great ! :) don't forget ask gentoo devs for listing in layman list :+1:

**Comment by pavlix** [comment]
@okias https://github.com/rafspiny/enlightenment-live/issues/1 :)
